### PR TITLE
Added links to integration tests in "Supported Services", closes #81

### LINKED ISSUE #81: Link to integration tests in Rusoto for each service
A link to the integration test file for each service on the supported services page should help folks find up to date examples of using each service. 👍 

EG: ECS would have a link to https://github.com/rusoto/rusoto/blob/master/integration_tests/tests/ecs.rs .

### PR DESCRIPTION
**Pull request by andizimmerer**
Cleaning up all the things for Hacktoberfest! :tada: 

Because this was long overdue and I think this would help some people (at least me, haha) I added all the links to integration tests to the "Supported Services" page (see #81).

Hope this helps a little!

I just used the name of the test file as the link name. This makes the new column quite big. Do you think some sort of unicode symbol would be better? :thinking: 

Furthermore, I noticed that the link for Import/Export [https://aws.amazon.com/documentation/importexport/](https://aws.amazon.com/documentation/importexport/) doesn't work anymore. Not sure what happened to this service.

And by the way: AWESOME WORK @matthewkmayer!! I haven't even heard of all services you wrote tests for :smile:

### CODE REVIEW

**Comment by matthewkmayer** [comment]
Thanks, I should be able to review this in the next few days. 👍

File: src/supported-aws-services.md [hunk 1]
```diff
@@ -1,160 +1,160 @@
...
 [Performance Insights](https://aws.amazon.com/rds/performance-insights/) | [rusoto_pi](https://crates.io/crates/rusoto_pi)
-[Polly](https://aws.amazon.com/polly/) | [rusoto_polly](https://crates.io/crates/rusoto_polly)
-[Pricing](https://aws.amazon.com/ec2/pricing/) | [rusoto_pricing](https://crates.io/crates/rusoto_pricing)
-[Redshift](https://aws.amazon.com/redshift/) | [rusoto_redshift](https://crates.io/crates/rusoto_redshift)
-[Rekognition](https://aws.amazon.com/rekognition/) | [rusoto_rekognition](https://crates.io/crates/rusoto_rekognition)
-[Resource Access Manager](https://docs.aws.amazon.com/cli/latest/reference/ram/index.html) | [rusoto_ram](https://crates.io/crates/rusoto_ram)
-[Resource Groups](https://docs.aws.amazon.com/ARG/latest/userguide/welcome.html) | [rusoto_resource_groups](https://crates.io/crates/rusoto_resource_groups)
-[Resource Groups Tagging](http://docs.aws.amazon.com/resourcegroupstagging/latest/APIReference/Welcome.html) | [rusoto_resourcegroupstaggingapi](https://crates.io/crates/rusoto_resourcegroupstaggingapi)
```
Comment (matthewkmayer) [inline src/supported-aws-services.md:126]: Nit: this one link is missing the `.rs` the others have.

**Comment by matthewkmayer** [comment]
Thanks for taking on this work! 😄 

Left one nit on there for making one line look like the others.

For your import/export question, looks like it's been made a flavor of Snowball: https://aws.amazon.com/snowball/disk/details/ .

How about we change the import/export service link to https://aws.amazon.com/snowball/disk/details/ ? 👍 

As for width, it looks okay to me. If you'd like, how about we try the emoji link in a different, followup PR?

**Comment by andizimmerer** [comment]
Wow, that's what I call a sharp eye! :grin: Really missed this one, but corrected it accordingly.
I also changed the AWS URL for import/export as you suggested.
If the width is okay for you, I would leave it as it is.

**Comment by matthewkmayer** [comment]
Thank you! 👍

**Pull request merged**